extensions/notebook/src/dialog/configurePython/pickPackagesPage.ts

```diff
@@ -76,7 +111,7 @@ export class PickPackagesPage extends BasePage {
 	public async onPageEnter(): Promise<void> {
 		let pythonExe = JupyterServerInstallation.getPythonExePath(this.model.pythonLocation, this.model.useExistingPython);
 		this.installedPackagesPromise = this.model.installation.getInstalledPipPackages(pythonExe);
-		this.installedPackages = undefined;
+		this.packageVersionMap = undefined;
```

Comment:
It'd be cleaner IMO if you just:

1. Created the map when declaring the var
2. Cleared it here instead of setting it to undefined
3. Chained a .then onto the call above that set the map. 
4. On line 142 you'd still await the promise but wouldn't need any other logic - the map would be populated after that returns.